### PR TITLE
Add PID file and Vagrant symlink to .gitignore

### PR DESCRIPTION
**Pull request by bjackman**

### CODE REVIEW

**Comment by mdigiorgio** [comment]
Since you are at it, we could probably add `vagrant` too, which appears as untracked file.

File: .gitignore [hunk 1]
```diff
@@ -5,3 +5,4 @@ results*
 *.json
 *.log
 tools/android-sdk-linux
+*.pid
```
Comment (derkling) [inline .gitignore]: A PID file should be generated only to track the ipython server, right?
Thus, maybe better:
```bash
ipynb/*.pid
```

Comment (bjackman) [inline .gitignore]: Sure, will amend.

**Comment by bjackman** [comment]
@mdigiorgio Ah yes, will add that too. Didn't do it initially because I wasn't sure if it was an artefact of my own setup.

**Comment by mdigiorgio** [comment]
Thanks, I guess it can be merged.

File: .gitignore [hunk 2]
```diff
@@ -6,3 +6,4 @@ results*
 *.log
 tools/android-sdk-linux
 ipynb/*.pid
+/vagrant
```
Comment (derkling) [inline .gitignore]: maybe without the initial "/" ?

Comment (bjackman) [inline .gitignore]: I think the `/` should be there - if there's a random file called `vagrant` anywhere else in the tree we shouldn't be ignoring it. 

However I've just noticed it's inconsistent with the PID one I just added and the android-sd-linux one. How about I stick a `/` at the beginning of all of them?

Comment (mdigiorgio) [inline .gitignore]: Yeah, actually it works both with and without the `/`.

**Pull request merged**